storage/aptosdb/src/lib.rs

```diff
@@ -1893,18 +1893,49 @@ impl DbWriter for AptosDB {
             )?;
 
             // Delete the genesis transaction
-            StateMerklePruner::prune_genesis(self.state_merkle_db.clone(), &mut batch)?;
             LedgerPruner::prune_genesis(
                 self.ledger_db.clone(),
                 self.state_store.clone(),
                 &mut batch,
             )?;
 
+            self.ledger_pruner
```

Comment:
Looking great. Just to understand the context and feel free to skip. If a node uses fast sync to a version, The pruner can only start from this version. This is no way to prune to an even earlier version?

Reply:
Currently we don't sync any data from pervious versions.